youtube_dl/extractor/extractors.py

```diff
@@ -1405,7 +1410,7 @@
     WebOfStoriesPlaylistIE,
 )
 from .weibo import (
-    WeiboIE, 
+    WeiboIE,
```

Comment:
Remove all unrelated changes.

Reply:
Fixed in c17500b33a70152a9598000ba5d47cb0cd0b3967